### ISSUE TITLE
Feature request: Follow times limit for "Follow someone else's following"

### ISSUE THREAD

**Issue by 46960** [InstaPy/InstaPy#2376]
Is there any way we could add a follow times limit for _following someone else's followers_? Same as already exists in follow by list basically. 

The reason I am asking is that instapy seems to always pick out the same 10 people when I run the script with the same accounts.  Would be cool if there was a way to have a follow limit and then skip that user and then go to follow a different user instead. People have been blocking me and wouldn't be surprised if I get reported too.

So basically what it would do for session.follow_user_followers would be to still go through someones followers, check how many times the given user has been followed before, and if its past the limit, it would show "user has already been followed more than n times", then skip them and go to the next one. 

Thoughts, remarks?
